mvc/src/main/java/nextstep/mvc/view/JsonView.java

```diff
@@ -1,13 +1,71 @@
 package nextstep.mvc.view;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-
+import java.io.IOException;
 import java.util.Map;
+import nextstep.mvc.exception.ViewException;
+import nextstep.web.support.MediaType;
+import org.apache.commons.lang3.ClassUtils;
 
 public class JsonView implements View {
 
+    private static final int MULTI_DATA_NUMBER = 2;
+    private static final int SINGLE_DATA_NUMBER = 1;
+    private static final String NO_DATA = "";
+
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
     @Override
-    public void render(final Map<String, ?> model, final HttpServletRequest request, HttpServletResponse response) throws Exception {
+    public void render(final Map<String, ?> model, final HttpServletRequest request,
+                       final HttpServletResponse response) {
+        response.setCharacterEncoding("utf-8");
+        response.addHeader("Content-Type", MediaType.APPLICATION_JSON_UTF8_VALUE);
+
+        writeData(response, toJson(model));
+    }
+
+    private String toJson(final Map<String, ?> model) {
+        if (model.size() >= MULTI_DATA_NUMBER) {
+            return toJsonWithObject(model);
+        }
+        if (model.size() >= SINGLE_DATA_NUMBER) {
```

Comment:
```suggestion
        if (model.size() == SINGLE_DATA_NUMBER) {
```
해당 부분은 SINGLE_DATA_NUMBER에만 해당될 수밖에 없는데 확실하게 부호를 이렇게 바꾸는 건 어떨까요?